qadence/engines/jax/differentiable_backend.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import annotations
+
+from collections import Counter
+from typing import Any
+
+from jax import Array
+
+from qadence.backend import Backend as QuantumBackend
```

Comment:
Not sure this is necessary, unless I'm mistaken. In `DifferentiableBackend`, it is `Backend`.